src/targeting.js

```diff
@@ -325,7 +325,10 @@ export function newTargeting(auctionManager) {
       Object.keys(targetingConfig).filter(customSlotMatching ? customSlotMatching(slot) : isAdUnitCodeMatchingSlot(slot))
         .forEach(targetId =>
           Object.keys(targetingConfig[targetId]).forEach(key => {
-            let valueArr = targetingConfig[targetId][key].split(',');
+            let valueArr = targetingConfig[targetId][key];
+            if (typeof valueArr === 'string') {
+              valueArr = valueArr.split(',');
+            }
             valueArr = (valueArr.length > 1) ? [valueArr] : valueArr;
```

Comment:
I don't know why this change is being made, but seems like it could cause an error if it gets to this line and valueArr is still a string, then valueArr.length is going to fail?